recipes/Momentum/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "Momentum" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/facebookincubator/momentum/archive/v{{ version }}.tar.gz
+    sha256: f37bb8288b873de3b804c367eadeec81c02bff2ceff470f35a9427ed4b426bfe
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_compatible(name, max_pin='x.x') }}
```

Not sure why the linter suggest `pin_compatible`, I have only ever seen `pin_subpackage` here.
```suggestion
    - {{ pin_subpackage(name, max_pin='x.x') }}
```

---

Hmm, I'm not sure. The linter still suggests using `pin_compatible`, and the documentation doesn't explain the differences between them:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#conda-build-specific-jinja2-functions